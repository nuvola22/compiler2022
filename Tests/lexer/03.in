_a
_
abc
AbC
____123
PrOgRaM