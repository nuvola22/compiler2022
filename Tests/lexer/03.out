1		1		Id			_a				_a
2		1		Id			_				_
3		1		Id			abc				abc
4		1		Id			AbC				AbC
5		1		Id			____123				____123
6		1		Keyword			Program				PrOgRaM
6		8		Eof							ï¿¿
